crates/sui-storage/src/mutex_table.rs

```diff
@@ -296,3 +333,47 @@ async fn test_mutex_table_bg_cleanup() {
         assert!(locked.is_empty());
     }
 }
+
+#[tokio::test]
```

Comment:
You probably want `#[tokio::test(flavor = "current_thread", start_paused = true)]`

Reply:
thanks, good point and done